OSLC4Net_SDK/scripts/vocab_gen.py

```
@@ -0,0 +1,157 @@
+        """)
+    else:
+        print("No properties found matching the criteria in the loaded data.")
+
+# --- 6. Handle potential errors ---
+except FileNotFoundError:
+    print(f"Error: Local file '{source_location}' not found.", file=sys.stderr)
+    print("Please ensure the file path is correct.", file=sys.stderr)
+    sys.exit(1) # Exit with a non-zero status code indicates an error
+except rdflib.exceptions.ParserError as pe:
+    print(f"Error parsing RDF data from '{source_location}': {pe}", file=sys.stderr)
+    print(f"Ensure the file is a valid '{source_format}' file.", file=sys.stderr)
```

Ruff (0.8.2)

Undefined name `source_format`

(F821)